onnxruntime/core/providers/cuda/cuda_execution_provider.cc

```diff
@@ -177,6 +177,10 @@ CUDAExecutionProvider::CUDAExecutionProvider(const CUDAExecutionProviderInfo& in
     }
   }
 
+#if defined(CUDA_VERSION) && CUDA_VERSION >= 11000  
```

Comment:
Any reason why this has to be CUDA 11 and above only ?

Reply:
CUDA Graph feature is only available for CUDA >= 11.0.

Reply:
https://github.com/microsoft/onnxruntime/pull/10423#discussion_r794252265

Reply:
https://developer.nvidia.com/blog/cuda-graphs/ - This also seems to be using CUDA 10.1 in the example ?

Reply:
Sorry, my bad. I will change it.